package.json

```diff
@@ -61,6 +61,7 @@
     "watchify": "^3.8.0"
   },
   "dependencies": {
+    "classnames": "^2.2.5",
```

Comment:
This is a neat little tool, but for a small library, we don't really need this as an additional dependency.